presto-raptor/src/main/java/com/facebook/presto/raptor/metadata/JdbcUtil.java

```diff
@@ -27,4 +38,27 @@ public static void enableStreamingResults(Statement statement)
             statement.unwrap(com.mysql.jdbc.Statement.class).enableStreamingResults();
         }
     }
+
+    public static JDBCType jdbcType(Type type)
```

Comment:
I don't think this should be moved here. It's not generic for JDBC (like enabling streaming), but rather is specific to how we encode columns for the index tables.